src/main/java/com/google/sps/servlets/YouTubeServlet.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.List;
+import java.util.Arrays;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.stream.Collectors;
+import com.google.sps.data.Video;
+
+@WebServlet("/yt")
+public class YouTubeServlet extends HttpServlet {
+
+    private Video video;
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I am not sure if this is intended -- where is "/yt" GET called?

Reply:
Yeah so far I haven't added this part yet. Probably will be added when we put the comments and captions features together.

Reply:
I see I see